Sprint-1/interpret/time-format.js

```diff
@@ -1,24 +1,30 @@
-const movieLength = 8784; // length of movie in seconds
+const movieLength = 999999; // length of movie in seconds
 
 const remainingSeconds = movieLength % 60;
 const totalMinutes = (movieLength - remainingSeconds) / 60;
+console.log("the total" + totalMinutes);
 
 const remainingMinutes = totalMinutes % 60;
 const totalHours = (totalMinutes - remainingMinutes) / 60;
 
 const result = `${totalHours}:${remainingMinutes}:${remainingSeconds}`;
 console.log(result);
 
-// For the piece of code above, read the code and then answer the following questions
+// For the piece of code above, read the ode and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//  6 variabls
 
 // b) How many function calls are there?
-
+//    1
 // c) Using documentation, explain what the expression movieLength % 60 represents
+//   The remainder (%) operator returns the remainder left over when one operand is divided by a second operand. It always takes the sign of the dividend.fb
```

Comment:
What does this particular remainder represent?

Reply:
Represents the remaining minutes that are not part of a full hour when the movie length is expressed in hours and minutes.